bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/container.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { createContainer } from 'meteor/react-meteor-data';
+
+import Service from './service';
+import PresentationUploader from './component';
+
+const PresentationUploaderContainer = props => (
+  <PresentationUploader {...props}>
+    {props.children}
```

Comment:
I couldn't find the case where this component has `children`. Probably this can be removed.